### PR TITLE
chore: Breakup large spec file to avoid chrome crash

### PR DESCRIPTION
**Pull request by BlueWinds**
### User facing changelog
No user facing changes

### Additional details
Cypress currently face out-of-memory crashes with Chrome in large spec files. Users run into this frequently, and we do as well. This PR doesn't address the root cause, but merely breaks up one of our larger test files to avoid the symptoms.

There are no changes to the tests in question - I just broke up a large file into two parts.

### PR Tasks

- [x] Have tests been added/updated?
- [n/a] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [n/a] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [n/a] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

**Review by rachelruderman — APPROVED** [review]
Nice idea, let's try it!

File: packages/app/cypress/e2e/runner/reporter.command_errors.cy.ts [hunk 1]
```diff
@@ -0,0 +1,512 @@
...
+  // Return scoped verify function with spec options baked in
+  return createVerify({ fileName: Cypress._.last(filePath.split('/')), hasPreferredIde, mode })
+}
+
+describe('errors ui', {
+  viewportHeight: 768,
+  viewportWidth: 1024,
+  // Limiting tests kept in memory due to large memory cost
+  // of nested spec snapshots
+  numTestsKeptInMemory: 1,
```
Comment (emilyrohrbough) [inline packages/app/cypress/e2e/runner/reporter.command_errors.cy.ts]: Might as well as just set this to 0.

Comment (BlueWinds) [reply packages/app/cypress/e2e/runner/reporter.command_errors.cy.ts]: Done.

Comment (BlueWinds) [reply packages/app/cypress/e2e/runner/reporter.command_errors.cy.ts]: The test file was still huge, so I like splitting it up even if we're not crashing chrome with `numTestsKeptInMemory: 0`.

**Comment by BlueWinds** [comment]
A couple of flaky tests in here unrelated to the changes; I don't want to block a flake fix on *other* flake fixes though, so if I get a second approval I will merge without a green build.

**Review by chrisbreiding — APPROVED** [review]

**Pull request merged**